src/azul/indexer/index_service.py

```diff
@@ -717,15 +720,19 @@ def _select_latest(self,
                         cur_bundle_uuid, cur_bundle_version, cur_entity = \
                             collated_entities.get(entity_id, (None, '', None))
```

Comment:
There should be an assertion somewhere that no bundle is created with an empty version, ensuring that this "bottom" version of '' works in all cases.